src/primitives/block.h

```diff
@@ -75,8 +78,7 @@ class CBlock : public CBlockHeader
 
 public:
     // Xpress Validation: (memory only)
-    // However Orphans or Missing transactions that have been re-requested must be verifed
-    // because their inputs have never been checked.
+    // Orphans, or Missing transactions that have been re-requested, are stored here.
```

Comment:
Let's start recovering doxygen auto-generated documentation.  This will save someone time later who has to go through all the code fixing everything. 

So please use //! to mark the comment so it is recognized as a docstring.